docs/administration/security.md

```diff
@@ -1,5 +1,627 @@
+        portalContainerName=portal
+        realmName=gatein-domain
+        password-stacking=useFirstPass;
+      org.exoplatform.services.security.j2ee.TomcatLoginModule required
+        portalContainerName=portal
+        realmName=gatein-domain;
+    };
+```
+
+In which:
+
+- `gatein-domain` is the **Realm name** which will be refered by applications. If you change this default name, you need to re-configure all the applications that use the Realm (listed later).
+
+- Two required **login modules** are: *org.gatein.sso.integration.SSODelegateLoginModule* and *org.exoplatform.services.security.j2ee.TomcatLoginModule*. The first, if authentication succeeds, will create an *Identity* object and save it into a shared state map, then the object can be used by the second.
+These are some login modules available in eXo Platform. Refer to `Existing login modules` to understand how they match the login scenarios.
+
+### Declaring JAAS Realm in eXo Platform
+
+#### In the Tomcat bundle
+
+- The default Realm is declared in the `$PLATFORM_TOMCAT_HOME/conf/jaas.conf` file. Its content is exactly the above example.
+
+- A \"security domain\" property in `$PLATFORM_TOMCAT_HOME/gatein/conf/exo.properties` (about this file, see `Configuration overview` needs to be set equal to the Realm name:
+
+```properties
+    exo.security.domain=gatein-domain
+```
+
+### List of applications using Realm
+
+If an application (.war) uses the Realm for authentication and
+authorization, it will refer to the Realm name with either of the
+following lines.
+
+- In `WEB-INF/jboss-web.xml`:
+
+    ``` xml
+    <security-domain>java:/jaas/gatein-domain</security-domain>
+    ```
+
+- In `WEB-INF/web.xml`:
+
+    ``` xml
+    <realm-name>gatein-domain</realm-name>
+    ```
+
+- In `META-INF/context.xml`:
+
+    ``` xml
+    appName='gatein-domain'
+    ```
+
+As mentioned above, if you change \"`gatein-domain`\", you need to
+re-configure all the applications that use the Realm to refer to the new
+Realm. Here is the list of webapps and the files you need to
+re-configure:
+
+**In the Tomcat bundle:**
+
+- `portal.war`: `/WEB-INF/jboss-web.xml`, `/WEB-INF/web.xml`,
+    `/META-INF/context.xml`.
+- `rest.war`: `/WEB-INF/jboss-web.xml`, `/WEB-INF/web.xml`.
+- `ecm-wcm-extension.war`: `/WEB-INF/jboss-web.xml`.
+- `calendar-extension.war`: `/WEB-INF/jboss-web.xml`.
+- `forum-extension.war`: `/WEB-INF/jboss-web.xml`.
+- `wiki-extension.war`: `/WEB-INF/jboss-web.xml`.
+- `ecm-wcm-core.war`: `/WEB-INF/jboss-web.xml`.
+
+::: tip
+
+The `.war` files are located under the `$PLATFORM_TOMCAT_HOME/webapps`
+folder.
+:::
+
+## Gadget proxy configuration
+
+In eXo Platform, you could allow gadgets to load remote resources.
+However, this could be a potential security risk, as it will make the
+Gadget deployed as an open web proxy. So, you can set up the anonymous
+proxy to accept or deny certain hosts by configuring the
+*ProxyFilterService*.
```

Comment:
I think this is outdated. We should remove

Reply:
yes removed